internal/provider/resource_aci_access_interface_override_test.go

```diff
@@ -0,0 +1,365 @@
+			{
+				Config: testConfigInfraHPathSReset,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "name", "host_path_selector"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotation", "orchestrator:terraform"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "description", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "name_alias", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "owner_key", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "owner_tag", ""),
+				),
+			},
+			// Import testing
+			{
+				ResourceName:      "aci_access_interface_override.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "name", "host_path_selector"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotation", "orchestrator:terraform"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "description", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "name_alias", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "owner_key", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "owner_tag", ""),
+				),
+			},
+			// Update with children
+			{
+				Config: testConfigInfraHPathSChildren,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotation", "orchestrator:terraform"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "description", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "name_alias", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "owner_key", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "owner_tag", ""),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotations.0.key", "key_0"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotations.0.value", "value_1"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotations.1.key", "key_1"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotations.1.value", "test_value"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "annotations.#", "2"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "relation_to_access_interface_policy_group.0.annotation", "annotation_1"),
+					resource.TestCheckResourceAttr("aci_access_interface_override.test", "relation_to_access_interface_policy_group.0.target_dn", "target_dn_1"),
```

Comment:
same issue here in relation to target, so the problem lies at the tesvars template